test/commonTaskInputs.test.ts

```diff
@@ -131,5 +131,21 @@ describe("getCredentials tests", () => {
       const result = getCredentials();
       result.cloudInstance.should.equal(variant.cloudInstance);
     });
-}
+  }
+
+  it("Returns incorrect cloud when cloud instance is explicitly passed as user input.", () => {
+    const tipTest = { endpoint: 'https://aurora.crm10.dynamics.com', cloudInstance: 'Tip1' }
```

Comment:
This repo is open source. Tip and Aurora are Microsoft internal resources.

Reply:
I can update this test, but this input already exists from earlier commits. 

See: https://github.com/microsoft/powerplatform-build-tools/blob/febdba36fbaa7aa79d16ddae1f8d0d0a65c866f8/test/commonTaskInputs.test.ts#:~:text=const%20endpointsTests%20%3D,%5D%3B

Reply:
Resolved in discussion